deploy/helm/kubecf/assets/operations/stacks.yaml

```diff
@@ -0,0 +1,178 @@
+{{- $_ := include "_config.lookup" (list $ "releases") }}
+{{- range $release_name, $release := $.kubecf.retval }}
+  {{- if $release.buildpack }}
+    {{- /* buildpack release conditions are either true or false, not complex expressions */}}
+    {{- if $release.condition }}
+    {{- /* *** Create/update "api" jobs for each buildpack (they already exist for default stack buildpacks) *** */}}
+- type: replace
+  path: /instance_groups/name=api/jobs/name={{ $release_name }}?/name
+  value: {{ $release_name | quote }}
+- type: replace
+  path: /instance_groups/name=api/jobs/name={{ $release_name }}/release?
+  value: {{ $release_name | quote }}
+- type: replace
+  path: /instance_groups/name=api/jobs/name={{ $release_name }}/properties?/quarks/bpm/processes
+  value: []
+    {{- else }}
+- type: remove
+  path: /instance_groups/name=api/jobs/name={{ $release_name }}?
+    {{- end }}
+  {{- end }}
+{{- end }}
+
+{{- if not $.Values.features.eirini.enabled }}
+  {{- range $stack_name, $stack := $.Values.stacks }}
+    {{- if $stack.enabled }}
+      {{- /* *** Create "diego-cell" job to setup rootfs for each stack *** */}}
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name={{ $stack_name }}-rootfs-setup?
+  value:
+    name: {{ $stack_name }}-rootfs-setup
+    release: {{ $stack_name }}
+    properties:
+      {{ $stack_name }}-rootfs:
+        trusted_certs:
+        - ((diego_instance_identity_ca.ca))
+      {{- if $.Values.features.credhub.enabled }}
```

Comment:
We are not quite ready to use `qcopy` because `image_list.rb` uses `bosh interpolate` to apply the ops files generated by `helm template`.

I'll leave this for later, when we do get around to organizing ops files. In a way, this is the very beginning of that effort.

I think the indented `{{- if $.Values.features.credhub.enabled }}` condition is the only thing that makes the new way slightly harder to read, but I find it still better to not create the cert list, and later add to it. So will leave as-is for now.